qa/evil-tests/src/test/java/org/elasticsearch/bootstrap/EvilJNANativesTests.java

```diff
@@ -80,4 +80,29 @@ public void testSetMaxSizeVirtualMemory() throws IOException {
         }
     }
 
+    public void testSetMaxFileSize() throws IOException {
+        if (Constants.LINUX) {
+            final List<String> lines = Files.readAllLines(PathUtils.get("/proc/self/limits"));
+            if (!lines.isEmpty()) {
```

Comment:
the `for` loop will prevent from reaching the assertions and returning if `lines` is empty?